framework_tvb/tvb/core/entities/file/simulator/monitor_h5.py

```diff
@@ -132,7 +132,7 @@ class BoldH5(MonitorH5):
     def __init__(self, path):
         super(BoldH5, self).__init__(path)
         self.period = Scalar(Bold.period, self)
-        self.hrf_kernel = EquationScalar(Bold.hrf_kernel, self)
+        self.equation = EquationScalar(Bold.equation, self)
```

Comment:
See the comments bellow. Why is this rename necessary?

Reply:
Answered this on an above comment.